### ISSUE TITLE
More monitors

### ISSUE THREAD

**Issue by Samq64** [ScratchAddons/ScratchAddons#4230]
### Is your request related to a problem? Please describe.

Some blocks don't have monitors, and they would be useful for some of them.

### Describe the solution you'd like

Make things like mouse y, days since 2000 have monitors.

### Describe alternatives you've considered

Using a variable in a forever loop.

**Comment by CST1229** [comment]
Note: Monitors for (most I think) other reporters _do_ work in vanilla Scratch, through hacked show/hide variable blocks.

**Comment by Oren-Lindsey** [comment]
> Note: Monitors for (most I think) other reporters _do_ work in vanilla Scratch, through hacked show/hide variable blocks. It also allows you to show and hide the existing non-variable monitors through code.

Yes they do. In fact you can even make monitors from any scratch block (they just don't return anything). See my project: https://scratch.mit.edu/projects/540817555/
